diplom.py

```diff
@@ -9,6 +9,20 @@ def parse(line):
     label = int (tokens[23])
     return (features, label)
 
+def Mean_Average_Precision (Predicted_probability_distribution, n_of_classes, list_of_classes, test_y, TEST_DATASET_SIZE):
```

Comment:
Also put arguments in separate lines:
def func_name(a,
                         b,
                         c):